### PR TITLE
OPS-1918 secret checks

### PR DESCRIPTION
**Pull request by say5**
https://liv-it.atlassian.net/browse/OPS-1918

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -75,6 +75,16 @@ steps:
     event:
     - push
 
+- name: secrets leaks test
+  pull: default
+  image: python:3.7.3-alpine3.8
+  commands:
+  - apk --update add bash git
+  - pip install -r requirements.txt
```
Comment (auraz) [inline .drone.yml]: development.txt

File: .drone.yml [hunk 2]
```diff
@@ -75,6 +75,16 @@ steps:
...
+  - trufflehog --exclude_paths trufflehog-exclude.txt --regex .
```
Comment (auraz) [inline .drone.yml]: trufflehog-exclude.txt is not present, probably it is ok?

File: Makefile [hunk 3]
```diff
@@ -11,6 +11,11 @@ help: ## display this help message
 	@perl -nle'print $& if m{^[0-9a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | \
 	awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
 
+setup: ## Setup local dev env.
+setup:
+		virtualenv -p python3 .env && . .env/bin/activate && \
+	  pip install -r requirements.txt
```
Comment (auraz) [inline Makefile]: development.txt

**Comment by auraz** [comment]
(not an action, but a thought) We run other scripts  (tox) without virtual env, maybe it is not needed for truffelhog.

File: .drone.yml (hunk 2 above)
Comment (say5) [reply .drone.yml]: present in root folder

**Pull request merged**